src/main/java/jp/ne/paypay/model/QRCodeResponse.java

```diff
@@ -120,12 +120,6 @@ public RedirectTypeEnum read(final JsonReader jsonReader) throws IOException {
   @SerializedName("authorizationExpiry")
   private Integer authorizationExpiry = null;
   
-  @SerializedName("amountescription")
```

Comment:
@javidlulu check this once in OPA API doc and confirm that this field is removed both from request and response?

Reply:
its not there in request and response https://www.paypay.ne.jp/opa/doc/v1.0/dynamicqrcode#operation/createQRCode.
Removed from SDK here.